treeherder/webapp/api/performance_serializers.py

```diff
@@ -1,8 +1,12 @@
+import decimal
+
 import six
 from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
 from rest_framework import (exceptions,
                             serializers)
 
+from treeherder.model import models
```

Comment:
I think it makes more sense to explicitly import the models that you need here rather than the whole model. It will be easier to catch mistakes and mispellings with the linter that way.